Tests/MapboxMapsTests/Annotations/Generated/CircleAnnotationManagerTests.swift

```diff
@@ -0,0 +1,200 @@
+import Foundation
+import XCTest
+@testable import MapboxMaps
+
+final class CircleAnnotationManagerTests: XCTestCase {
+
+    var manager: CircleAnnotationManager!
+    var style: MockStyle!
+    var displayLinkCoordinator: DisplayLinkCoordinator!
+    var id = UUID().uuidString
+    var annotations = [CircleAnnotation]()
+
+    override func setUp() {
+        super.setUp()
+
+        style = MockStyle()
+        displayLinkCoordinator = MockDisplayLinkCoordinator()
```

Comment:
Would also be nice to crystallize in tests:
* that the manager subscribes/unsubscribes to the display link coordinator
* that annotations data driven properties are merged manager's layerProperties correctly
* that unset layer/properties are reset with default values
* that annotations are translated to GeoJSONObject correctly
* that handleQueriedFeatureIds passes/not passes a notification to it's delegate

Reply:
All added in newest commit 👍